client/reader/stats.js

```diff
@@ -77,5 +79,9 @@ export function recordUnfollow( url ) {
 }
 
 export function recordTrack( eventName, eventProperties ) {
+	const subCount = SubscriptionStore.getTotalSubscriptions();
```

Comment:
One potential risk is that, in #3418, we switch to incrementing `totalSubscriptions` in the store and no longer use the subscription count from the first page of the response. This count mean that getTotalSubscriptions() reports 50, 100, 150.... as the subs get loaded in.